unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -48,29 +48,31 @@ export const MembersContent = ({ account, network, lockAddresses }: Props) => {
       <Head>
         <title>{pageTitle('Members')}</title>
       </Head>
-      {account && (
-        <BrowserOnly>
-          <Account network={network} account={account} />
-          <Filters>
-            Show{' '}
-            <Filter
-              value={MemberFilters.ACTIVE}
-              current={filter}
-              setFilter={value => setFilter(value)}
-            />
-            <Filter
-              value={MemberFilters.ALL}
-              current={filter}
-              setFilter={value => setFilter(value)}
+      <Authenticate>
+        {account && (
```

Comment:
Same

Reply:
same it uses `account.address` below and since react does not know that this cannot be rendered if account is null, it complains.